lib/arena.rb

```diff
@@ -1,2 +1,58 @@
+require_relative "../lib/gladiator"
+
 class Arena
+  attr_reader :name
+  attr_reader :gladiators
+
+  def initialize (name)
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator(gladiator)
+    if @gladiators.count < 2
+      self.gladiators << gladiator
```

Comment:
What's the reasoning behind using `self.gladiators` here vs. `@gladiators`? In this case both end up doing the same thing - just curious!